packages/app/src/components/search/SearchPage.tsx

```diff
@@ -70,9 +67,7 @@ const SearchPage = () => {
       <Content>
         <Grid container direction="row">
           <Grid item xs={12}>
-            <Paper className={classes.bar}>
```

Comment:
Bumming? Do you mean bumping? this is the example app version, not mandatory but probably better if users do apply these same changes in their own apps.

Reply:
Oops, I mean collision; it was a typo!
I'm not sure how many adopters use this example, and since this is out of our control, I thought it would be good to add a disclaimer in the changeset to recommend they apply these changes in case they use the example.

Reply:
Updating the changeset, let me know if the wording is correct.